py-polars/polars/io.py

```diff
@@ -206,6 +206,24 @@ def read_csv(
     --------
     scan_csv : Lazily read from a CSV file or multiple files via glob patterns.
 
+    Examples
+    --------
+    >>> import polars as pl
```

Comment:
"import polars as pl" is done automatically for all doctests! So you can leave this out.